packages/core/src/Stores/traders-hub-store.js

```diff
@@ -368,6 +370,12 @@ export default class TradersHubStore extends BaseStore {
         this.setCombinedCFDMT5Accounts();
     }
 
+    get financial_restricted_countries() {
+        const { financial_company, gaming_company } = this.root_store.client.landing_companies;
+
+        return financial_company?.shortcode === 'svg' && !gaming_company;
```

Comment:
```suggestion
        return (financial_company?.shortcode === 'svg' || financial_company?.shortcode === 'maltainvest') && !gaming_company;
```

we have to check for Malta too